### DISCUSSION TITLE
Wazuh Version and possibility of putting it in Downloads with OSQuery

### DISCUSSION THREAD

**Discussion by gr8matt** [Security-Onion-Solutions/securityonion#1606] [Unsupported Versions] [answered]
All,

How do I see what version of Wazuh is running to ensure I download the correct Wazuh agents?  The new documentation is great and makes it clear that the agent and server needs to be running the same version but it doesn't spell out how to check the version of Wazuh that is running.   

Also, it's great that the team added osquery downloads directly into SOC.  Is there a reason you decided against adding Wazuh agents there as well?  It definitely makes the airgap install easier.

Absolutely amazed the work that was put into SO 2!!!  Great job!
Matt

**Comment by dougburks** [comment, marked as answer]
Hi @gr8matt !

> How do I see what version of Wazuh is running to ensure I download the correct Wazuh agents? The new documentation is great and makes it clear that the agent and server needs to be running the same version but it doesn't spell out how to check the version of Wazuh that is running.

```
sudo docker exec -it so-wazuh dpkg -l |grep wazuh
ii  wazuh-api      3.13.1-1     amd64        Wazuh API is an open source RESTf
ii  wazuh-manager  3.13.1-1     amd64        Wazuh helps you to gain security 
```
This matches the 3.13.1 version mentioned in the documentation:
https://docs.securityonion.net/en/2.3/wazuh.html#adding-agents


> Also, it's great that the team added osquery downloads directly into SOC. Is there a reason you decided against adding Wazuh agents there as well? It definitely makes the airgap install easier.

Mike created an issue for that:
https://github.com/Security-Onion-Solutions/securityonion/issues/1609

> Absolutely amazed the work that was put into SO 2!!! Great job!

Thanks!
